_posts/2020-07-31-kubeflow-1.1-blog-post.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Kubeflow 1.1 improves ML Workflow Productivity, Isolation & Security, and GitOps"
+description: "Kubeflow 1.1 Blog Post"
+layout: post
+toc: false
+comments: true
+image: images/some_folder/your_image.png
+hide: false
+search_exclude: true
+categories: [fastpages, jupyter]
+metadata_key1: metadata_value1
+metadata_key2: metadata_value2
```

Comment:
```suggestion
image: images/logo.png
hide: false
categories: [release, official]
```

Reply:
Cleanup your front matter

Reply:
@jbottum looks like you just copied and pasted my example.  That was just a toy example of front matter.  I tried to clean it up for you here with this suggested commit